api/data/filters.sql

```diff
@@ -3,51 +3,51 @@ DELETE FROM public.page_filters;
 -- Sections in the survey
 
 -- Section 1 - General Information
-INSERT INTO public.page_filters VALUES ('sector', 'Agriculture;Forestry;Both');
-INSERT INTO public.page_filters VALUES ('type-of-stakeholder', 'Farmer/agricultural producers;Forester;Forest owner;Forest operator;Forest product processor;Farming association;Farming cooperative;Forestry association;Forest industry association;Trade association;NGO/Advisory Group;Data association/organisation/coalition;Data provider;Infrastructure provider;Platform provider;Service/Information provider;Digital technology provider;Research institutes and research networks;National and European networks');
-INSERT INTO public.page_filters VALUES ('location-country-region', 'Austria;Belgium;Bulgaria;Croatia;Cyprus;Czechia;Denmark;Estonia;Finland;France;Germany;Greece;Hungary;Ireland;Italy;Latvia;Lithuania;Luxembourg;Malta;Poland;Portugal;Romania;Slovakia;Slovenia;Spain;Sweden;The Netherlands');
-INSERT INTO public.page_filters VALUES ('primary-area-of-operation-in-agriculture', 'Crop cultivation- grains;Crop cultivation- vegetables;Crop cultivation- legumes;Crop cultivation- fruits;Plant propagation;Livestock farming - meat;Livestock farming- dairy;Livestock farming- other;Mixed Farming (crops and animal);Agricultural machinery and equipment services;Crop services (monitoring);Farm management services;Post-harvest handling services;Other');
-INSERT INTO public.page_filters VALUES ('primary-area-of-operation-in-forestry', 'Reforestation;Forest conservation - thinning, pruning, weed & pest control;Felling;Transportation of logs;Non-Timber Forest Products (NTFPs);Forest Fire Management;Forestry inventory and mapping;Wildlife management;Other');
-INSERT INTO public.page_filters VALUES ('organic-farming-operation', 'Yes;No;Don''t know');
+INSERT INTO public.page_filters VALUES ('sector', 'Agriculture;Forestry;Both;N/A');
+INSERT INTO public.page_filters VALUES ('type-of-stakeholder', 'Farmer/agricultural producers;Forester;Forest owner;Forest operator;Forest product processor;Farming association;Farming cooperative;Forestry association;Forest industry association;Trade association;NGO/Advisory Group;Data association/organisation/coalition;Data provider;Infrastructure provider;Platform provider;Service/Information provider;Digital technology provider;Research institutes and research networks;National and European networks;N/A');
```

Comment:
Overall, I think the best solution would if we change this approach to dynamically retrieve the filters based on the actual data. 

We can then implement smart filtering, because as of now, it's quite easy to end up with no-data in the platform. I think this is a hit for the usability

cc @andresgnlez